bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/util/DebuggerRuntimeHelperUtils.java

```diff
@@ -0,0 +1,158 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.runtime.internal.util;
+
+import io.ballerina.runtime.api.PredefinedTypes;
+import io.ballerina.runtime.api.Runtime;
+import io.ballerina.runtime.api.async.Callback;
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BFuture;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.internal.scheduling.Scheduler;
+import io.ballerina.runtime.internal.scheduling.Strand;
+import io.ballerina.runtime.internal.util.exceptions.BallerinaException;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.concurrent.CountDownLatch;
+import java.util.function.Function;
+
+/**
+ * This class contains the set of runtime helper util methods to support debugger expression evaluation.
+ * <p>
+ * These utils methods must be class-loaded into the program JVM to evaluate
+ * <ul>
+ *  <li> function
+ *  <li> object method
+ *  <li> remote call action
+ *  <li> wait action
+ *  </ul>
+ * invocations using the debugger expression evaluation engine.
+ *
+ * @since 2.0.0
+ */
+@SuppressWarnings("unused")
+public class DebuggerRuntimeHelperUtils {
```

Comment:
Shall we rename to DebuggerRuntima and move to separate package inside internal

Reply:
Renamed and moved in 0c17327